### ISSUE TITLE
Control metadata rendering (project and date) inside todoist query

### ISSUE THREAD

**Issue by bzimor** [jamiebrynes7/obsidian-todoist-plugin#130]
**Is your feature request related to a problem? Please describe.**
I want to see tasks in my daily note with `today` filter. In this case rendering date is redundant. It is same with my project note, I want to see tasks with `#project` filter, so there is no point to show project name in metadata.

**Describe the solution you'd like**
There can be an view in todoist query to choose what kind of metadata items can be shown in task list. For example:

```
```todoist
{
"name": "Project tasks",
"filter": "#MyProject",
"view": ["date", "tag"]
}
```
**Additional context**
Todoist plugin v1.8.1

**Comment by BrandonHarrisonCode** [comment]
As a workaround for now, you can disable showing the date and project globally in Obsidian in the project options for this plugin. But a per-query way of organizing this would absolutely be nice.

**Comment by BrandonHarrisonCode** [comment]
As a workaround for now, you can disable showing the date and project globally in Obsidian in the project options for this plugin. But a per-query way of organizing this would absolutely be nice.

**Comment by jebeaudoin** [comment]
+1 for enabling metadata rendering on a per-query basis. This would probably satisfy requests to hide the "Today" label on tasks due Today that others have asked for.

**Comment by jamiebrynes7** [comment]
This is now released in [v1.12.0](https://github.com/jamiebrynes7/obsidian-todoist-plugin/releases/tag/1.12.0) 🎉
